six/modules/c++/cphd/wscript

```diff
@@ -9,3 +9,4 @@ def build(bld):
     modArgs = globals()
     modArgs['VERSION'] = bld.env['SIX_VERSION']
     bld.module(**modArgs)
```

Comment:
Can you update this to install the CPHD schema too? See `six/modules/c++/six.sidd/wscript` for an example